gatk-wdl/Dockerfile

```diff
@@ -0,0 +1,42 @@
+
+FROM ubuntu@sha256:cc4755a9f2f76ca73055da11f1bdc01d65ce79202a68b041c67c32770c71954b
+
+LABEL description="Dockerfile with requirements for GATK test wdl pipeline" \
+version="1.0.0" \
+maintainer="eva@lifebit.ai"
+
+USER root
+
+RUN apt-get update -y \
+  && apt-get install curl -y \
+  && apt-get install bzip2 -y\
+  && apt-get install wget -y \
+  && apt-get install procps -y \
+  && apt-get install git -y \
+  && rm -rf /var/lib/apt/lists/*
+
+
+#Install cromwell instance
+
+RUN wget https://github.com/broadinstitute/cromwell/releases/download/33.1/cromwell-33.1.jar
+RUN wget https://github.com/broadinstitute/cromwell/releases/download/47/cromwell-47.jar
```

Comment:
Remove these 2 lines, as Cromwell (and Nextflow) is designed to use the container.